codersdk/templates.go

```diff
@@ -513,3 +513,28 @@ func (c *Client) StarterTemplates(ctx context.Context) ([]TemplateExample, error
 	var templateExamples []TemplateExample
 	return templateExamples, json.NewDecoder(res.Body).Decode(&templateExamples)
 }
+
+type InvalidatePrebuildsResponse struct {
+	InvalidatedPresets []string `json:"invalidated_presets"`
```

Comment:
Maybe it would make sense to also send the template name and version in the response to better description of which template/version was actually invalidated?